go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/unrolled/secure
+
+go 1.12
+
+require github.com/codegangsta/negroni v1.0.0
```

Comment:
Let's remove this one... it's only used within the tests, not an actual requirement for the package. It can be installed in the travis `before_install` like the static check tool.

Reply:
With go module dependencies used by test are a part of the `go.mod`, yes I know it's weird.

It's not really recommend to manage manually the `go.mod` file.

Take a look to `go mod tidy`

Reply:
Well thats just crazy... but makes sense :P